src/test/resources/features/tags/tags_tests.feature

```diff
@@ -1,11 +1,27 @@
 Feature: Create Tags
 
   @CreateTag
-  Scenario: As a user I want to create a Tag for tracking an objective
+  Scenario: As a user I want to create a Tag
     Given I set the tags with request endpoint to /space/{space_id}/tag
     When I set the tags body with following values:
       | name   | APIV2   |
       | tag_fg | #ffffff |
       | tag_bg | #3c9d9b |
       And I execute the POST request for tags
     Then I verify that the response status is 200
+
+  @CreateTag
+  Scenario: As a user I want to create a Tag with only name
+    Given I set the tags with request endpoint to /space/{space_id}/tag
+    When I set the tags body with following values:
+      | name   | APIV10   |
+    And I execute the POST request for tags
+    Then I verify that the response status is 200
+
+  Scenario: As a user I want to verify that it is not possible to create a Tag without name
```

Comment:
this scenario doen-t need a tag'?

Reply:
Yes because it is negative and doesnt need pre conditions